environment/src/main/java/com/sequenceiq/environment/parameters/validation/validators/AwsParameterProcessor.java

```diff
@@ -10,30 +10,32 @@
 import com.sequenceiq.environment.environment.service.NoSqlTableCreationModeDeterminerService;
 import com.sequenceiq.environment.parameters.dao.domain.S3GuardTableCreation;
 import com.sequenceiq.environment.parameters.dto.AwsParametersDto;
+import com.sequenceiq.environment.parameters.dto.ParametersDto;
 import com.sequenceiq.environment.parameters.service.ParametersService;
 
 @Component
-public class AwsParameterValidator {
+public class AwsParameterProcessor {
```

Comment:
What was the problem with the processor? The result still Validation but the class and method are Processor

Reply:
It does not just validate, it also computes if we create new dynamo table name or use existing one, and saves it in the repository. We agreed with @bergerdenes that this is the good name.

Reply:
It could also be named as a decorator, because it is not really processing more rather decorates the state/behaviour of the environment.